odl/contrib/fom/test/test_supervised.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tests for supervised FoMs."""
+
+import numpy as np
+import pytest
+import odl
+import odl.contrib.fom
+from odl.util.testutils import simple_fixture
+
+space = simple_fixture('space', [odl.rn(3),
+                                 odl.uniform_discr(0, 1, 10)])
+
+def test_mean_squared_error(space):
```

Comment:
Given the recent issue #1136 an additional check that the normalized variants do the right thing would be good. That means checking `0 <= result_normalized <= 1` for random inputs, `(x, x)` and `(x, -x)` which are often best and worst case, resp.